diagnostics-app/src/app/telemetry/telemetry-card/telemetry-card-content/telemetry-card-content.component.ts

```diff
@@ -0,0 +1,31 @@
+// Copyright 2023 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+/**
+ * @fileoverview Defines the CardContent Component in Telemetry.
+ * Imported by telemetry.module.ts
+ */
+
+import { Component, Input } from '@angular/core';
+import { KeyValue } from '@angular/common';
+
+@Component({
+  selector: 'app-telemetry-card-content',
+  templateUrl: './telemetry-card-content.component.html',
+  styleUrls: ['./telemetry-card-content.component.css']
+})
+export class TelemetryCardContentComponent {
+  @Input({ required: true }) info: Object | undefined;
+  @Input({ required: true }) layer: number = 0;
+
+  public originalOrder = (a: KeyValue<number, string>, b: KeyValue<number, string>): number => {
```

Comment:
I added a comment to explain the use of the function. And I think it's better to maintain the original order since it conforms to what users will expect to understand the data easily.